style-guide/styles/wds-components/_page-header.scss

```diff
@@ -0,0 +1,132 @@
+.wds-page-header {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+	background-position: right center;
+	background-repeat: no-repeat;
+	background-size: auto 100%;
+	height: 105px;
+	position: relative;
+	z-index: 1;
+
+	@media #{$wds-breakpoint-l-down} {
+		height: 115px;
+	}
+
+	&::before {
+		content: '';
+		display: block;
+		height: 100%;
+		position: absolute;
+		width: 100%;
+		z-index: -1;
+	}
+
+	&__wordmark {
+		float: left;
+		height: 65px;
+		margin: 20px 0 20px 20px;
+		max-width: 250px;
+		width: auto;
+
+		@media #{$wds-breakpoint-l-down} {
+			height: 49px;
+			margin: 20px 0 0 20px;
+			max-width: 175px;
+			width: auto;
+
+			img {
+				max-height: 49px;
+				width: auto;
+			}
+		}
+	}
+
+	&__top-container {
+		display: flex;
+		height: 59px;
+
+		@media #{$wds-breakpoint-l-down} {
+			height: 69px;
+		}
+	}
+
+	&__wordmark-text {
+		flex: 1;
+		font-size: $wds-typescale-size-plus-3;
+		font-weight: 600;
+		line-height: 1;
+		margin: 24px 0 0 18px;
+		overflow: hidden;
+		white-space: nowrap;
```

Comment:
Why won't we add `text-overflow: ellipsis;` here? It's an easy and fast solution and will look nicer than truncated letter.